src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -491,6 +493,26 @@ public function validateSchema(\DOMDocument $dom)
     }
 
     /**
+     * Validates an alias.
+     *
+     * @param \DOMElement
+     */
+    private function validateAlias(\DOMElement $alias, $file)
+    {
+        foreach ($alias->attributes as $name => $node) {
+            if (!in_array($name, array('alias', 'id', 'public'))) {
+                @trigger_error(sprintf('Using the attribute "%s" is deprecated for alias definition "%s" in "%s". Allowed attributes are "alias", "id" and "public". The XmlFileLoader object will raise an exception instead in Symfony 4.0 when detecting an unsupported attribute.', $key, $alias->getAttribute('id'), $file), E_USER_DEPRECATED);
```

Comment:
I'd improve the second sentence slightly in all these deprecation messages and say "*The XmlFileLoader object will raise an exception in Symfony 4.0, instead of silently ignoring unsupported attributes.*".

Currently the message says "*will raise an exception instead*" but it doesn't say instead of what.